CHANGELOG.rst

```diff
@@ -1,3 +1,42 @@
+.. _changelog-unreleased:
+
+Unreleased
+==========
+
+ **🚨 Attention:**
+
+    The turbodbc dependency was pinned to *4.5.4* due to issues with newer versions.
+
+    Failing tests in the SQLA compliance test suite, in regard to the turbodbc dialect
+    won't be addressed until explicitly required/requested by users.
+
+    Note: It is also very likely that turbodbc support will be dropped in future versions.
+
+✨ Added
+--------
+
+* Add websocket based dbapi2 compliant database driver
```

Comment:
Do you usually add ticket or PR numbers to the changelog?

Reply:
Could be done but there is no single ticket for this one this is rather a preparation (first step) needed to implement the following [ticket](https://github.com/exasol/sqlalchemy-exasol/issues/105).